_community/google-groups.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Google Groups - Managing List Membership
+layout: text-optimized-width
+
+---
+
+# {{ page.title }}
+
+This documents how to join and leave a Google Group via webpage and email.
+
+If you have any issues following the guides below please reach out to us at [support@osg-htc.org](mailto:support@osg-htc.org) with the email you would like subscribed and the list you would like to subscribe too.
+
+##### Contents
```

Comment:
Is there no way to have this as a sidebar? There is a lot of wasted space on this page, IMO.